pkg/controllers/execution/execution_controller.go

```diff
@@ -302,8 +303,9 @@ func (c *Controller) syncToClusters(clusterName string, work *workv1alpha1.Work)
 	syncSucceedNum := 0
 	for _, manifest := range work.Spec.Workload.Manifests {
 		workload := &unstructured.Unstructured{}
-		util.MergeLabel(workload, util.ManagedByKarmadaLabel, util.ManagedByKarmadaLabelValue)
 		err := workload.UnmarshalJSON(manifest.Raw)
+		util.MergeLabel(workload, util.ManagedByKarmadaLabel, util.ManagedByKarmadaLabelValue)
```

Comment:
Remove this line then, duplicate with https://github.com/karmada-io/karmada/blob/5135e8fea33cc2fadf9d8c7d4ce7211023991b80/pkg/controllers/binding/common.go#L144.